### ISSUE TITLE
Http reuest for api calling in ionic1 app is not working

### ISSUE THREAD

**Issue by NileshSystematix** [ionic-team/cordova-plugin-ionic-webview#60]
Hello Guys,

I am added this plugin in my updated ionic1 iOS app as I am facing some UI issue in iOS 11. But after adding this plugin my api calling by http request is not working throughout the app and it is always coming in err scenario. Before adding this plugin my api are working fine witht the header.

So can you please help me what to pass in header if it is the issue of headers in api. Please check my below api request.

 var req8 = {
            method: 'POST',
            url: "api url",
            params: {
            "device_type": "iOS",
            "device_token": token
            
            },
            headers: {
            'Content-Type': 'application/x-www-form-urlencoded'
            }
            }

 $http(req8).then(function(res) {
}, function myError(err) {
                             $ionicLoading.hide();
                             var alertPopup = $ionicPopup.alert({
                                                                template: 'Connection Error. Please reload.'
                                                                });
                             }
)

**Comment by vinicoliveira** [comment]
Did you solve the problem? I have the some problem here

**Comment by antoine92190** [comment]
What is the error you are getting? Is it a CORS issue?

**Comment by vinicoliveira** [comment]
Yes. It is a CORS issue. But if I remove the plugin everything works fine.

**Comment by antoine92190** [comment]
Then the solution would be to use the HTTP plugin: https://ionicframework.com/docs/native/http/
as mentioned here: https://ionicframework.com/docs/wkwebview/#i-cant-implement-cors
You would need to adapt your code a bit to make http requests using this plugin.

**Comment by vinicoliveira** [comment]
Does it work with v1?

**Comment by antoine92190** [comment]
I guess, you will have to give it a try.

**Comment by NileshSystematix** [comment]
@vinicoliveira 

Are you facing the problem of padding in header or footer? Just replace this line with the applied meta line in index.html

**Comment by vinicoliveira** [comment]
I am sorry...padding?
If I use this plugin combined with angular $http I get some CORS errors. Like status -1. If I remove this plugin everything works fine. 

As @antoine92190 said...I will try the HTTP plugin. I will have to change all my app but it is ok =/

**Comment by NileshSystematix** [comment]
@vinicoliveira I think in ionic1 app we don't need to install this plugin. I am also facing the the issue of $http while calling the api. So just implement this it will work fine.
